### PR TITLE
Onboard Dance installer repo

### PR DESCRIPTION
**Pull request by xinredhat**

### CODE REVIEW

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-master-rhtap-installer-shellcheck

**Comment by xinredhat** [comment]
/pj-rehearse pull-ci-redhat-appstudio-rhtap-installer-master-rhtap-installer-shellcheck

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  commands: redhat-appstudio-rhtap-installer-e2e-test-commands.sh
+  credentials:
+  - mount_path: /usr/local/ci-secrets/redhat-appstudio-qe
+    name: ci-secrets
+    namespace: test-credentials
+  from: e2e-test-runner
+  resources:
+    requests:
+      cpu: 100m 
+
```
Comment (srivickynesh) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-ref.yaml:13]: remove these empty lines

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/static-checks/yamllint/redhat-appstudio-rhtap-installer-static-checks-yamllint-commands.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+pwd
+ls -l
+yamllint version
+# yamllint -c "$CONFIG_FILE"
```
Comment (srivickynesh) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/static-checks/yamllint/redhat-appstudio-rhtap-installer-static-checks-yamllint-commands.sh:10]: Yamllint config is missing? we enable it later?

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/static-checks/shellcheck/redhat-appstudio-rhtap-installer-static-checks-shellcheck-commands.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+pwd
+ls -l
+find . -type f -name "*.sh" -print0 | xargs -r -0 shellcheck
+
+sleep 600
```
Comment (srivickynesh) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/static-checks/shellcheck/redhat-appstudio-rhtap-installer-static-checks-shellcheck-commands.sh:11]: Do we need 10 minutes of explicit sleep here?

**Comment by srivickynesh** [comment]
lgtm other than few question posted

**Comment by flacatus** [comment]
Against what cluster are you testing?

**Comment by xinredhat** [comment]
> Against what cluster are you testing?
 konflux cluster pools

**Pull request merged**